### PR TITLE
added the liquid button prototype

### PR DESCRIPTION
**Pull request by HakimSkaik**
Added the liquid connect button, but we need to determine the background so I could finalize the connect background. Also, the button needs to be 'warmed up' before it works.. not sure why so perhaps you could take a look.

### CODE REVIEW

**Comment by danielle-dsouza** [comment]
Holyyyy you did a ton of work on this - wow! Do you mind adding a quick screen recording?

**Comment by HakimSkaik** [comment]
https://user-images.githubusercontent.com/98859282/190277437-17c7aad7-076d-49c9-9f67-3072f5731ffb.mp4

Here's a preview of the button, the more you hover on it, the more interactive it gets

**Comment by danielle-dsouza** [comment]
Let's wait before merging this - I'd like to run it by UX and the rest of the team first. But good job!

File: src/sections/Home/Home.tsx [hunk 1]
```diff
@@ -84,9 +85,10 @@ function Home() {
             </li>
           </ol>
           <div className="SectionAssets">
-            <Button colorBorder inline>
+            {/* <Button colorBorder inline>
               Connect
-            </Button>
+            </Button> */}
```
Comment (danielle-dsouza) [inline src/sections/Home/Home.tsx:90]: Let's delete the commented code

**Review by danielle-dsouza — APPROVED** [review]
We got the A-okay from UX!

**Pull request merged**